setup.py

```diff
@@ -12,15 +12,29 @@
 import platform
 import re
 import ast
+import sys
 from setuptools import find_packages, setup
 from setuptools.extension import Extension
 from setuptools.command.build_ext import build_ext as _build_ext
 
+# numpy is not required with help or information display options
+if any(
+    arg.startswith('--') and arg not in [
+        '--verbose',
+        '--quiet',
+        '--dry-run',
+        '--no-user-cfg',
+        '--command-packages'] for arg in sys.argv):
+    setup_requires = []
+else:
+    setup_requires = ['numpy >= 1.9.2']
 
 # Bootstrap setup.py with numpy
 # Huge thanks to coldfix's solution
 # http://stackoverflow.com/a/21621689/579416
-class build_ext(_build_ext):
+ 
+ 
+ class build_ext(_build_ext):
```

Comment:
Can you revert the changes to these lines? It is causing the Travis build to error:

```
Processing /home/travis/build/biocore/scikit-bio
    Complete output from command python setup.py egg_info:
    Traceback (most recent call last):
      File "<string>", line 20, in <module>
      File "/tmp/pip-7RadPw-build/setup.py", line 37
        class build_ext(_build_ext):
                                   ^
    IndentationError: unindent does not match any outer indentation level
```